source/gui/addonStoreGui/controls/messageDialogs.py

```diff
@@ -403,7 +403,7 @@ def _setupUI(self):
 		mainSizer.Add(sHelper.sizer, border=BORDER_FOR_DIALOGS, flag=wx.ALL)
 		self.Sizer = mainSizer
 		mainSizer.Fit(self)
-		self.CentreOnScreen()
+		self.Maximize()
```

Comment:
I don't think this dialog needs to be maximized, that's quite disruptive for a notification popup. Instead it should be sized appropriately

Reply:
OK, now it's centered, not maximized.